### DISCUSSION TITLE
Updated Notepad

### DISCUSSION THREAD

**Discussion by IsthisSid** [codepharm/motiv8m3#23] [Announcements]
Hiya @codepharm/codepharmers ! I tried to summarize everything we reviewed this past weekend. Please check my work and feel free to make any changes. ^_^
![FunnyCodeGif](https://user-images.githubusercontent.com/72266833/182161369-cd3a8160-e5f6-499e-bbd4-54d478687e29.gif)
That is my look of fear each time haha

**Comment by codeTrackLift** [comment]
Awesome summary of what we covered yesterday, great job!

**Comment by codeTrackLift** [comment]
Comments I **JUST** got from a Sr dev on PR naming:

- Start the PR name with the ticket/issue number in brackets
- Use plain text to describe what the PR changes
- Be more detailed than the branch name, but still concise
